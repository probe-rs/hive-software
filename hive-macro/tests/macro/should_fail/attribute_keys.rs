#[hive_macro::hive_test(wrong_key = 1)]
fn test() {}

fn main() {}
