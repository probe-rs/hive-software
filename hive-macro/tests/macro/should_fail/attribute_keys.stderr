error: Failed to parse supplied macro attribute keys.

         = help: The following attributes and datatypes are allowed in this context: 'order = usize', 'should_panic = bool'
         = note: example: #[hive_test(order = 1, should_panic = true)]

 --> tests/macro/should_fail/attribute_keys.rs:1:25
  |
1 | #[hive_macro::hive_test(wrong_key = 1)]
  |                         ^^^^^^^^^
